src/PagerView.tsx

```diff
@@ -55,12 +55,19 @@ import { getViewManagerConfig, PagerViewViewManager } from './PagerViewNative';
 
 export class PagerView extends React.Component<PagerViewProps> {
   private isScrolling = false;
+  private animationFrameRequestId = 0;
```

Comment:
```suggestion
  private animationFrameRequestId;
```

Probably id can be also a 0, so it is better to not initialize it by 0

Reply:
The [docs](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame) specifically say it can't be 0, but I have changed it anyway.

> This is a non-zero value, but you may not make any other assumptions about its value.

Reply:
> The [docs](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame) specifically say it can't be 0, but I have changed it anyway.
> 
> > This is a non-zero value, but you may not make any other assumptions about its value.

This doc refers to web behaviuor. I am not sure, how it is implemented in RN. 
But anyway thank you